recipes/filelock/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "0.1" %}
+
+package:
+  name: filelock
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/dmfrey/FileLock 
```

Comment:
Please use a tarball instead of cloning. Also, pick a particular tag/commit to use. If there are no tags, please ask the author in an issue to add one.